aspnetcore/migration/mvc.md

```diff
@@ -3,18 +3,189 @@ title: Migrate from ASP.NET MVC to ASP.NET Core MVC
+1. Select the *_Layout.cshtml* file, then select **Add**, replacing the existing file.
+
+In the ASP.NET Core project, open the *_Layout.cshtml* file. Make the following changes to match the completed code shown below:
+
+Update the Bootstrap CSS inclusion to match the completed code below:
+
+1. Replace `@Styles.Render("~/Content/css")` with a `<link>` element to load *bootstrap.css* (see below).
+1. Remove `@Scripts.Render("~/bundles/modernizr")`.
+
+The completed replacement markup for Bootstrap CSS inclusion:
+
+```html
+<link rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+    crossorigin="anonymous">
+```
+
+Update the jQuery and Bootstrap JavaScript inclusion to match the completed code below:
+
+1. Replace `@Scripts.Render("~/bundles/jquery")` with a `<script>` element (see below).
+1. Replace `@Scripts.Render("~/bundles/bootstrap")` with a `<script>` element (see below).
+
+The completed replacement markup for jQuery and Bootstrap JavaScript inclusion:
+
+```html
+<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+    integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+```
+
+The updated *_Layout.cshtml* file is shown below:
+
+[!code-cshtml[](mvc/samples/3.x/Views/Shared/_Layout.cshtml?highlight=7-10,40-42)]
+
+View the site in the browser. It should now render with the expected styles in place.
+
+## Configure bundling and minification
+
+ASP.NET Core is compatible with several open-source bundling and minification solutions such as [WebOptimizer](https://github.com/ligershark/WebOptimizer) and other similar libraries.  ASP.NET Core does not provide a native bundling and minification solution. For information about how to configure bundling and minification, see [Bundling and Minification](../client-side/bundling-and-minification.md).
+
+## Solve HTTP 500 errors
+
+There are many problems that can cause an HTTP 500 error message that contains no information on the source of the problem. For example, if the *Views/_ViewImports.cshtml* file contains a namespace that doesn't exist in your project, you'll get a HTTP 500 error. By default in ASP.NET Core apps, the `UseDeveloperExceptionPage` extension is added to the `IApplicationBuilder` and executed when the configuration is *Development*. This is detailed in the following code:
+
+[!code-csharp[](mvc/samples/3.x/Startup.cs?highlight=30-35)]
+
+ASP.NET Core converts unhandled exceptions in a web app into HTTP 500 error responses. Normally, error details aren't included in these responses to prevent disclosure of potentially sensitive information about the server. See **Using the Developer Exception Page** in [Handle errors](../fundamentals/error-handling.md) for more information.
```

Comment:
> See **Using the Developer Exception Page** in [Handle errors](../fundamentals/error-handling.md) for more information.

See https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-3.1#developer-exception-page

`#developer-exception-page` is the key.